src/main/java/io/chazza/advancementapi/AdvancementAPI.java

```diff
@@ -2,18 +2,20 @@
 
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
 import lombok.Builder;
 import lombok.Getter;
 import lombok.Singular;
-import net.md_5.bungee.api.chat.BaseComponent;
+import net.md_5.bungee.api.chat.TextComponent;
```

Comment:
Isn't it bad to import bungee cord api as then the project becomes complient to it?